src/Dropdown.js

```diff
@@ -41,9 +41,11 @@ class Dropdown extends Component {
   renderTrigger () {
     const { trigger } = this.props;
 
+    const className = typeof trigger.className === 'string' ? `${trigger.className} dropdown-button` : 'dropdown-button';
+
     return React.cloneElement(trigger, {
       ref: (t) => (this._trigger = `[data-activates=${this.idx}]`),
-      className: 'dropdown-button',
+      className,
```

Comment:
So what spawned all this is that I was passing `<Button />` as the trigger, however since this is all inside a form the button press would always trigger submit.  If I add `e.preventDefault()` to the button handler then it stops the dropdown from working (I could have tried changing the button type probably). So, I am passing regular dom elements which don't have `props` so we can't assume that trigger is a React Element, though it is a node. Would you suggest we set up conditions for both?